### ISSUE TITLE
Where would I find the text colour of the control?

### ISSUE THREAD

**Issue by Wagnerp** [ComponentFactory/Krypton#72]
Hi,

This is a really simple question, where would I find the text colour of the control? Also, where can I find the caption/text property of a KryptonGroupBox control so that I can change it?

**Comment by FreeFlyingEnchilada** [comment]
The text and other things that generally will not change depending on the state of the object (normal/disabled/selected/etc) are within the Values property. So for a KryptonGroupBox the title is in: box.Values.Heading.

For things that do change based on the state of the object (like text color), you'll find it in a variety of different override groups. For instance kryptonGroupBox.StateCommon.Content.ShortText exposes a variety of different elements you can change about the text. And of course you'd just use StateDisabled or StateNormal or whatever if you just wanted to override parts of an object when it was in the given State.

**Comment by Wagnerp** [comment]
Hi @FreeFlyingEnchilada 

Thanks for the response. It wasn't well documented in the help file. I wonder if it's missing some pages, as doing a search for kryptonGroupBox returns nothing.

**Comment by FreeFlyingEnchilada** [comment]
Yeah, I also noticed that. I've found the sample projects to be really helpful. For instance in this case there is "KryptonGroupBox Examples 2015" in Source\Krypton Toolkit Examples\KryptonGroupBox Examples.
